mpi-proxy-split/lower-half/copy-stack.c

```diff
@@ -204,10 +212,11 @@ dbg_auxv_ptr_addr = dest_curr_stack;
   // We copied __environ to dest_stack earlier.  So, we're pointing to stack.
   // We use the "padded" variable so that it will be 16-byte aligned.
   dest_curr_stack -= env_ptr_size;
-  memcpy(dest_curr_stack, env_ptr_addr, env_ptr_size);
+  memcpy(dest_curr_stack, new_env_ptrs, env_ptr_size);
```

Comment:
In the line above `memcpy()`, let's add a comment:
```
  // We can't just copy from env_ptr_addr, to dest_curr_stack because
  // env_ptr_addr is an array of pointers to the _old_ stack.  We need
  // new_env_ptrs, an array pointing to the _new_ stack.
```